metis-normalization/src/test/java/eu/europeana/normalization/dates/extraction/dateextractors/CenturyDateExtractorTest.java

```diff
@@ -18,13 +21,14 @@ class CenturyDateExtractorTest {
   private static final CenturyDateExtractor CENTURY_DATE_EXTRACTOR = new CenturyDateExtractor();
 
   void extract(String input, String expected, DateNormalizationExtractorMatchId dateNormalizationExtractorMatchId) {
```

Comment:
suggestion: rename to `assertCenturyDateExtractor`

Reply:
Done. For consistency I've kept it similar to other tests.